docs/advanced_tutorials.rst

```diff
@@ -0,0 +1,13 @@
+.. _advanced-tutorials:
```

Comment:
Can we make this a section in `tutorials.rst` rather than creating another file?

Reply:
Done!